core/src/main/java/nu/yona/server/subscriptions/service/PinResetRequestService.java

```diff
@@ -73,7 +73,7 @@ public void sendPinResetConfirmationCode(UUID userId)
 		sendConfirmationCodeTextMessage(user, pinResetConfirmationCode);
 	}
 
-	@Transactional
+	@Transactional(dontRollbackOn = PinResetRequestConfirmationException.class)
```

Comment:
Does this fix a bug?

Reply:
Without this, the number of attempts for failed confirmations wouldn't be saved.